### DISCUSSION TITLE
No audio and FPS drop (only through internet)

### DISCUSSION THREAD

**Discussion by cecc83** [blakeblackshear/frigate#14061] [Ask A Question]
Hello everyone, I have a full HD camera at 25 fps (Annke AC800). It seems to work well in local environment, but when i exit lan with my phone and i try to reach it, fps drop and there's no audio. Internet speed test show +8 Mb/s in upload and router stream traffic when i call the cam is around 2 Mb/s (camera has variable bitrate, max 4096 kb/s). I have tried to reach camera both directly in home assistant UI through frigate card and with tailscale in frigate UI, in this last scenario sound start few seconds and so mute (and speaker icon on right top disapper). It seems to switch automatically to MJPEG and in docker log i see request for /api/<camera_name>/latest.webp (or /api/<camera_name>/latest.jpg). If I set max bitrate of camera from 4094 kb/s to 256 kb/s or 512 kb/s sound play well for minutes and image is smoothly, but quality is very low. From 1024 kb/s and above the problem persist. It's only a internet performance issue or I can do something in configuration to solve the problem?

**Comment by hawkeye217** [comment]
Sounds like you're just saturating your bandwidth. Use this calculator to determine how much bandwidth you're using with different settings: https://www.jvsg.com/storage-bandwidth-calculator/

**Reply by cecc83** [reply]
Thank you for your advice, with 1920x1080 resolution, medium quality, 25 fps the calculated bandwith should be around 3174 kb/s.
When i watch camera with my phone on external network this is upload chart (even lower):
![image](https://github.com/user-attachments/assets/cb23191c-f56d-4839-a09e-60c3af1178d5)
so much lower than the limit, and from frigate mini pc this is my internet speed test:
`speedtest-cli --simple
Ping: 14.792 ms
Download: 67.68 Mbit/s
Upload: 7.78 Mbit/s`
my camera configuration:
![image](https://github.com/user-attachments/assets/9cdee09a-0ff4-4ab8-85aa-30c1d5dd268f)
